src/components/measureEditor/MeasureEditor.tsx

```diff
@@ -249,15 +250,10 @@ const MeasureEditor = () => {
           : [];
 
       let valuesetsErrors = null;
-      if (
-        translationResults.library?.valueSets?.def !== null &&
-        tgt &&
-        tgtValue &&
-        tgtValue !== ""
-      ) {
+
+      if (translationResults.library?.valueSets?.def !== null && isLoggedIn) {
```

Comment:
Why is the logged in check here and the Code validation check is in `useTerminologyServiceApi`?

Reply:
@jkotanchik-SB Sorry just saw the comment, this is to validate value sets, only logged user can do it.